installer/install-clj-kondo-lsp.cmd

```diff
@@ -1,3 +1,3 @@
 @echo off
-
-curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v2020.05.09/clj-kondo-lsp-server-2020.05.09-standalone.jar
+New-Variable -Name "VERSION" -Value "2020.05.09"
+curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v$VERSION/clj-kondo-lsp-server-$VerSION-standalone.jar
```

Comment:
```suggestion
curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v%VERSION%/clj-kondo-lsp-server-%VERSION%-standalone.jar
```

Reply:
I'm good with that, not a pro on powershell ><